docs/guide_about_files_generated.md

```diff
@@ -0,0 +1,305 @@
+Guide
```

Comment:
this is still not quite correct - it should be a reStructuredText document (*.rst) that can be built by sphinx. Do you need help with this?

Also, somehow it needs to fit in with the rest of the developer's documentation.